packages/realm-web-integration-tests/harness/index.ts

```diff
@@ -40,6 +40,9 @@ const EXPECTED_ISSUES = [
   /was set with `SameSite=None` but without `Secure`/,
   // User / "refresh invalid access tokens" is posting an invalid token
   "Failed to load resource: the server responded with a status of 401 (Unauthorized)",
+  // User / "can be deleted" is deleting an invalid token and attempting a failing login
+  "Failed to load resource: the server responded with a status of 401 (Unauthorized)",
+  "Failed to load resource: the server responded with a status of 401 (Unauthorized)",
```

Comment:
i would like some explanation about what this data structure is doing and why there are multiples of the string in it.

Reply:
It seems like the `checkIssues` function could be refactored using the array methods `some` and `includes` and that would probably remove the need to have the same "expected issues" listed in this array.

Reply:
Does the documentation comment above clear things up?

> A list of expected errors and warnings logged to the console

It's there to ensure our SDK are not triggering unexpected errors or warnings in the console.

The reason the same message is in the array more than once, is because we don't want to allow an unlimited number of messages (even if they have the same message), as they might be caused by some other (potentially new) regression.

If we had to refactor this, we should probably move these messages into the tests somehow. Make them a property on the tests or something:

```typescript
it("prints an error", function() {
  this.expectedErrors([ "the error" ]);
  console.error("the error");
});
```

Reply:
So I've looked into this deeper and believe that it would be much more useful to refactor the tests with the proposed syntax from Kraen:

> If we had to refactor this, we should probably move these messages into the tests somehow. Make them a property on the tests or something:
```typescript
it("prints an error", function() {
  this.expectedErrors([ "the error" ]);
  console.error("the error");
});
```
In my quick study, I found that removing one of the elements from the array caused the test harness to fail, without any hint of why, other than that some log was received at some point that it wasn't expecting.  Putting this into the test itself would help make this more maintainable in the future.

Reply:
I have created https://github.com/realm/realm-js/issues/4323 to track the work.

Reply:
It seems the main blocker in fixing this is that it's unclear how to pass the log messages from the headless chrome into the test suites.  Perhaps one would have to setup some sort of channel to transmit the log messages from the log event to the mocha client.  In any case, not so trivial